scripts/superdesk-search/directives/ItemList.js

```diff
@@ -1550,6 +1550,13 @@ export function ItemList(
                 },
 
```

Comment:
I would make a function here:
```js
deselectAll: () => listComponent.setState({selected: null}),
```

And use that below in both places. Instead of `$rootScope.$broadcast('item:unselect');` and further down:
```js
scope.$on('item:unselect', this.deselectAll);
```
That broadcast can involve many side-effects later, and this way you can use a local function instead of broadcasting again via root.